packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html

```diff
@@ -1,78 +1,87 @@
-                  placeholder={placeholder}
-                  type="search"
-                  onkeyup={onKeyup}
-                  onfocus={onFocus}
-                  onblur={onBlur}
-                  aria-autocomplete="list"
-                  aria-controls="listbox-id"
-                  autocomplete="off"
-                  role="textbox"
-                />
-                <template if:false={isQueryEmpty}>
-                  <lightning-button-icon
-                    onclick={clearInput}
-                    icon-name="utility:close"
-                    variant="bare"
-                    class="slds-input__icon slds-input__icon_right input__icon_cursor"
-                    size="medium"
-                    name="Clear"
-                    title="Clear"
-                    aria-label="Clear"
-                    alternative-text="Clear"
-                  ></lightning-button-icon>
+  <template if:true={isStandalone}>
+    <c-quantic-search-interface engine-id={standaloneEngineId} skip-first-search disable-state-in-url>
+      <div class="slds-form-element">
+        <div class="slds-form-element__control">
+          <div class="slds-combobox_container">
+            <div
+              class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click"
+              aria-expanded="true"
+              aria-haspopup="listbox"
+              role="combobox"
+            >
+              <div class="searchbox__container">
+                <div class={searchBoxContainerClass} role="none">
+                  <template if:true={withoutSubmitButton}>
+                    <lightning-icon
+                      class="slds-icon slds-input__icon slds-input__icon_left"
+                      icon-name="utility:search"
+                      size="x-small"
+                    ></lightning-icon>
+                  </template>
+                  <input
+                    id="combobox-with-button-id"
+                    class="slds-input slds-combobox__input searchbox__input"
+                    placeholder={placeholder}
+                    type="search"
+                    onkeyup={onKeyup}
+                    onfocus={onFocus}
+                    onblur={onBlur}
+                    aria-autocomplete="list"
+                    aria-controls="listbox-id"
+                    autocomplete="off"
+                    role="textbox"
+                  />
+                  <template if:false={isQueryEmpty}>
+                    <lightning-button-icon
+                      onclick={clearInput}
+                      icon-name="utility:close"
+                      variant="bare"
+                      class="slds-input__icon slds-input__icon_right input__icon_cursor"
+                      size="medium"
+                      name="Clear"
+                      title="Clear"
+                      aria-label="Clear"
+                      alternative-text="Clear"
```

Comment:
We don't localize those strings ?

Reply:
Oh yes we do! haha i just wrapped the thing in a template tag. didn't even review the original markup. on it